tests/Fixer/ClassNotation/OrderedTypesFixerTest.php

```diff
@@ -143,6 +145,42 @@ public static function provideFixCases(): iterable
 }
 ',
         ];
+
+        yield 'sort alpha, null none' => [
```

Comment:
It would be better to move those new tests to different provider and to not add `@requires PHP 8.0` to `testFixCases()`, because catching multiple exceptions' types were added in 7.1 and should be tested also on still supported 7.4 🙂

Reply:
Done.